insights/tests/client/apps/test_compliance.py

```diff
@@ -232,3 +235,17 @@ def test_build_oscap_command_append_tailoring_path(config):
     compliance_client = ComplianceClient(config)
     expected_command = 'oscap xccdf eval --profile aaaaa --tailoring-file tailoring_path --results output_path xml_sample'
     assert expected_command == compliance_client.build_oscap_command('aaaaa', 'xml_sample', 'output_path', 'tailoring_path')
+
+@patch("insights.client.config.InsightsConfig")
+def test__get_inventory_id(config):
+    compliance_client = ComplianceClient(config)
+    compliance_client.conn._fetch_system_by_machine_id = lambda: []
+    with raises(SystemExit):
+        compliance_client._get_inventory_id()
+
+    compliance_client.conn._fetch_system_by_machine_id = lambda: [{}]
+    with raises(SystemExit):
+        compliance_client._get_inventory_id()
+
+    compliance_client.conn._fetch_system_by_machine_id = lambda: [{'id': '12345'}]
+    assert compliance_client._get_inventory_id() == '12345'
```

Comment:
I decided to test this private method, since I've mocked it out of all other tests.